recipes/casadi/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "casadi" %}
+{% set version = "3.1.1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/casadi/casadi/archive/{{ version }}.tar.gz
+  sha256: 335eab9270a06d18134dafe34b5a21fe26fe6926611d7f960d826d82e851abef
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [np==111]
```

Comment:
as above, we run into errors on travis when we try to build this 6 times... (running over the 1h30 build time limit). So following @scopatz's suggestion, we skip all but the most recent numpy.